forc-plugins/forc-crypto/src/args.rs

```diff
@@ -6,8 +6,20 @@ use std::{
     path::Path,
 };
 
+forc::cli_examples! {
+    [ Hashes data passing as an argument with SHA256 => crypto "sha256" "test" ]
```

Comment:
```suggestion
    [ Hashes an argument with Sha256 => crypto "sha256" "test" ]
```

I don't really understand the phrasing here but I can infer based on usage